src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1584,7 +1584,14 @@ static Library()
         {
             {
                 BuiltinFunctionsCore.AbsT,
-                StandardErrorHandlingTabularOverload<NumberValue>(BuiltinFunctionsCore.AbsT.Name, SimpleFunctionImplementations[BuiltinFunctionsCore.Abs])
+                StandardErrorHandlingTabularOverload<NumberValue>(BuiltinFunctionsCore.AbsT.Name, StandardErrorHandling<NumberValue>(
+                    BuiltinFunctionsCore.Abs.Name,
+                    expandArguments: NoArgExpansion,
+                    replaceBlankValues: ReplaceBlankWithZero,
+                    checkRuntimeTypes: ExactValueTypeOrBlank<NumberValue>,
+                    checkRuntimeValues: DeferRuntimeValueChecking,
+                    returnBehavior: ReturnBehavior.AlwaysEvaluateAndReturnResult,
+                    targetFunction: Abs))
```

Comment:
It is unfortunate that we have to go back to the full StandardErrorHandling here; we can probably add some logic to the StandardErrorHandlingTabularOverload to deal with blanks.